parser/parser.y

```diff
@@ -1298,6 +1298,10 @@ DefaultValueExpr:
 	{
 		$$ = &ast.FuncCallExpr{FnName: model.NewCIStr("CURRENT_TIMESTAMP")}
 	}
+|	NowSym '(' NUM ')'
+	{
+		$$ = &ast.FuncCallExpr{FnName: model.NewCIStr("CURRENT_TIMESTAMP")}
```

Comment:
In mysql, num must be the same with number specified in timestamp type: 
```
create table t (CREATED_ timestamp(3) not null DEFAULT CURRENT_TIMESTAMP(3));
```
and the following statement returns error because 2 != 3 :

```
create table t (CREATED_ timestamp(3) not null DEFAULT CURRENT_TIMESTAMP(2));
```
Despite MySQL have this grammar and strict check, I believe NUM is actually redundant information and can be ignored. we get the same result in TiDB.